tests/dopamine/replay_memory/circular_replay_buffer_test.py

```diff
@@ -638,6 +666,33 @@ def testLoad(self):
     self.assertEqual(memory.add_count, self._test_add_count)
     self.assertAllClose(memory.invalid_range, self._test_invalid_range)
 
+  def test_memory_is_locked(self):
```

Comment:
Best practices are that each unit test tests precisely one behavior.  I think http://go/unit-test-practices#behavior-testing is a relevant go link.

There are a number of behaviors being tested in this function.  I can't tell how well this best-practice is followed in the rest of the project.  If it's not so much, I think it's well-motivated to push back on this recommendation.

Reply:
My point is that each test should test a single behavior, not a set of behaviors through a workflow.  I'm not a testing expert, but this is feedback I've gotten when submitting my own CLs when applying for readability.

Reply:
:+1: